examples/replica/outputs.tf

```diff
@@ -9,7 +9,7 @@ output "id" {
 
 output "replica_id" {
   description = "Postgresql read-only replica instance id"
-  value       = module.read_only_replica_postgresql_db.id
+  value       = module.read_only_replica_postgresql_db[0].id
```

Comment:
I think it should return all the ids of the replicas instead of the first one

Reply:
Yes, will add that.